Database/Corrections/cataQuestFixes.lua

```diff
@@ -9203,10 +9203,14 @@ function CataQuestFixes.Load()
         [28856] = { -- The Sands of Silithus
             [questKeys.nextQuestInChain] = 8280,
         },
-        [28857] = { -- Hero's Call: Blasted Lands!
-            [questKeys.exclusiveTo] = {28673,28867},
+        [28857] = { -- Blasted Lands: The Other Side of the World
+            [questKeys.exclusiveTo] = {28673,28867}, -- Not sure if true. Needs double checking.
             [questKeys.nextQuestInChain] = 25710,
         },
+        [28858] = { -- Blasted Lands: The Other Side of the World
+            [questKeys.exclusiveTo] = {},
```

Comment:
Are you sure it is not exclusive to one of the other breadcrumb quests? That would mean you are able to accept/complete all three of them 🤔